src/graph_gui.cpp

```diff
@@ -622,6 +623,8 @@ struct OperatingProfitGraphWindow : BaseGraphWindow {
 	OperatingProfitGraphWindow(WindowDesc *desc, WindowNumber window_number) :
 			BaseGraphWindow(desc, WID_CV_GRAPH, STR_JUST_CURRENCY_SHORT)
 	{
+		this->draw_dates = true;
```

Comment:
What if we initialize `draw_dates` to `true` by default? Now it kinda gives the suggestion it can remain uninitialized, which is a bit weird to read.

But that makes me wonder what is going on with the other variables. Only to find out they too aren't explicitly set; just in some cases. Which all feels a bit ... odd, to me. But that is not your code.

Can't remember where OpenTTD stands on C++ initializers, but I personally really prefer to have them set explicitly in the definitions when possible. So `bool draw_dates = true`. And set it to false for the exception.

Curious what others think.

Reply:
Sure, makes sense to me. Should we continue explicitly set it to true for the windows which use it, just for visibility, or let it use the default?

Reply:
The latter, I would say. No need to set things again if they are already set.

Mainly what bothers me, is that some variables are set (like `draw_dates` for many Windows, but others only for one window.

Feels weird .. haven't bothered to actually look at the code what is going on, I have to admit. But feels like someone was a bad boy years ago, and we are now suffering :D